### ISSUE TITLE
Debugger stopped working after last update, no stack trace?

### ISSUE THREAD

**Issue by whillas** [microsoft/ptvsd#1972]
## Environment data
From the about dialog
Version: 1.40.2
Commit: f359dd69833dd8800b54d458f6d37ab7c78df520
Date: 2019-11-25T14:54:40.719Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Linux x64 5.3.7-050307-generic snap

Ubunut 18.04
Python 3.6.8-64bit

### launch.json
```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Run All Tests",
            "type": "python",
            "request": "launch",
            "module": "pytest"
        }
    ]
}
```

## Expected behaviour

Run test in debugger

## Actual behaviour

gives me the following output
```
...
APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL=true VSCODE_CLI=1 ELECTRON_NO_ATTACH_CONSOLE=1 NO_AT_BRIDGE=1 CHROME_DESKTOP=code-url-handler.desktop 'VSCODE_NLS_CONFIG={"locale":"en-gb","availableLanguages":{},"_languagePackSupport":true}' VSCODE_NODE_CACHED_DATA_DIR=/home/alex/.config/Code/CachedData/f359dd69833dd8800b54d458f6d37ab7c78df520 VSCODE_LOGS=/home/alex/.config/Code/logs/20191206T130433 VSCODE_IPC_HOOK=/run/user/1000/snap.code/vscode-d32d3ff9-1.40.2-main.sock VSCODE_PID=8850 AMD_ENTRYPOINT=vs/workbench/services/extensions/node/extensionHostProcess PIPE_LOGGING=true VERBOSE_LOGGING=true VSCODE_IPC_HOOK_EXTHOST=/tmp/vscode-ipc-45ac6eb7-3ae3-4d36-b119-40520381f570.sock VSCODE_HANDLES_UNCAUGHT_ERRORS=true VSCODE_LOG_STACK=false PATH=/home/alex/dev/concourse_server/__venv__/bin:/home/alex/dev/go/bin:/usr/local/go/bin:/home/alex/.local/bin:/home/alex/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin PYTHONIOENCODING=UTF-8 PYTHONUNBUFFERED=1 /home/alex/dev/concourse_server/__venv__/bin/python /home/alex/.vscode/extensions/ms-python.python-2019.11.50794/pythonFiles/ptvsd_launcher.py --default --client --host localhost --port 45991 -m pytest 
============================================================================ test session starts =============================================================================
platform linux -- Python 3.6.8, pytest-5.2.1, py-1.8.0, pluggy-0.13.0
rootdir: /home/alex/dev/imagr/concourse_server
collected 0 items / 1 errors                                                                                                                                                 INTERNALERROR> Traceback (most recent call last):
Traceback (most recent call last):


During handling of the above exception, another exception occurred:



During handling of the above exception, another exception occurred:



During handling of the above exception, another exception occurred:


Terminated
```

As you can see its not reporting the problem :-?

## Steps to reproduce:

It was working until recently. I think the last auto update might have coursed it...?
Why would it not be showing the track trace?
Works fine from a terminal running `python -m pytest`

**Comment by int19h** [comment]
Is it specific to pytest, or are you seeing the same when trying to run e.g. a simple hello world type app?

**Comment by whillas** [comment]
yup, seems to run `print("Hello world")` just fine. I've setup another project and it works fine for pytest. I guess its something to do with the project and pytest. I reinstalled the later and no cigar. I can start to remove things and see if it starts to run again...

**Comment by whillas** [comment]
Ok, looks like it has something to do with `zeromq`. Even though none of my tests are testing code related to it...?

**Comment by int19h** [comment]
Ultimately, this will get fixed once we ship #1706.
